scss/_patterns_navigation.scss

```diff
@@ -238,6 +244,29 @@ $lightness-threshold: 70;
         margin-top: $sp-medium !important;
         order: 1;
       }
+
+      &--toggle {
```

Comment:
Please avoid creating new class names like this, it makes it hard to understand what the class name really is. I don't know if it is `p-navigation--toggle` or `p-navigation-search--toggle`? Just write the full class name here.